_core/general/resredir.php

```diff
@@ -22,11 +17,11 @@
     error(S_NOTHREADERR, $dest);
 }
 
-$redirect = DATA_SERVER . BOARD_DIR . "/" . RES_DIR . "/" . (($resto == 0) ? $no : $resto) . PHP_EXT . '#' . $no;
+$redirectect = DATA_SERVER . BOARD_DIR . "/" . RES_DIR . "/" . (($redirect['resto'] == 0) ? $redirect['no'] : $redirect['resto']) . PHP_EXT . '#' . $redirect['no'];
 
-echo "<META HTTP-EQUIV='refresh' content='0;URL=$redirect'>";
+echo "<META HTTP-EQUIV='refresh' content='0;URL=$redirectect'>";
 
-if ($resto == "0") {
+if ($redirect['resto'] == "0") {
```

Comment:
`== "0"`? The most recent comparison checked it as an integer.

Pay no attention to the previous version of this line.